composer.json

```diff
@@ -24,22 +24,22 @@
         "php":                             "^7.4.7",
         "ext-json":                        "*",
         "dflydev/fig-cookies":             "^2.0.1",
```

Comment:
Note to self: since this PR will require a new MAJOR, we can bump dflydev/fig-cookies to v3